Gopkg.lock

```diff
@@ -228,70 +228,73 @@
 
 [[projects]]
   branch = "master"
-  digest = "1:109f5bac6ea240c7be934080356ccc4c7f0f455a2ab4b417282971503451b13e"
+  digest = "1:e9e6c4c2ae7f624a6eb1b768c582c05f200041de51faf5151b8e074eae917b57"
   name = "github.com/giantswarm/errors"
   packages = ["tenant"]
   pruneopts = "UT"
-  revision = "2f0151a3dfe0a682dbb3ce710068941c4b24e2b8"
+  revision = "2640113be13f2eec85fe785ed74418a1a2852e5d"
 
 [[projects]]
   branch = "master"
-  digest = "1:6087475a619c7ca89e5425b20db6b750e0d729d3523cf6e44a2619a5f26b7501"
+  digest = "1:c3c652e8e4ed4d3ec0fcca1cae82d5384a31fb38b06945941d531a9dd023c9c3"
   name = "github.com/giantswarm/helmclient"
   packages = ["."]
   pruneopts = "UT"
-  revision = "a5e3404ad0fc68dafb8ecb6e83c19b8e5ba69d5f"
+  revision = "55ea69a27c1e3bba18122269f75dfe304f6ce3cc"
 
 [[projects]]
   branch = "master"
-  digest = "1:ebcdcc8456d443d01881843f52a9adb3da342b2265a94e60425f6b74e224b2a8"
+  digest = "1:e7e6a0fd3834a59ca94f05ac446f1d07642035602c89b86898833b6ed66170da"
   name = "github.com/giantswarm/k8sclient"
-  packages = ["."]
+  packages = [
+    ".",
+    "k8scrdclient",
```

Comment:
Using k8scrdclient from k8sclient, instead of operatorkit, which allowed to drop cyclic dependency on operatorkit